bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py

```diff
@@ -339,6 +334,14 @@ def get_cost_of_kick_relative(self, x: float, y: float, direction: float, kick_l
         return self.get_cost_of_kick(pose.pose.position.x, pose.pose.position.y, d, kick_length, angular_range)
 
     def get_cost_of_kick(self, x: float, y: float, direction: float, kick_length: float, angular_range: float) -> float:
+        """
+        Returns the cost of the kick at the given position
+        :param x: Field coordiante in the x direction
+        :param y: Field coordiante in the y direction
+        :param direction: The direction of the kick
+        :param kick_length: The length of the kick
+        :param angular_range: The angular range of the kick
+        """
```

Comment:
Same here

Reply:
x and y are relative to the robot, but not in the field coordinate system in get_cost_of_kick_relative anymore?

Reply:
I don't know what you mean. In this function everything is in the field coordinate system.